### DISCUSSION TITLE
Resolution

### DISCUSSION THREAD

**Discussion by Zafoue** [UglyToad/PdfPig#684] [Q&A]
Hello , 
any methode to get the dpi of the pdf file  ( the idea is to save an image with this resolution using  GhostscriptRasterizer )
Thanks

**Comment by petrsorfa** [comment]
Offhand there doesn't seem to be a way of getting the DPI from the document itself - but the assumption is that it is 72dpi.
Each page has a separate width and height which seem to match that in the PDFs I've processed.

If you use the UglyToad.PdfPig.Rendering.Skia extension, using  document.GetPageAsPng (index, scale, color), you can set the scale to multiply the DPI, so setting the scale option to 2 will cause the rendered PNG to be at 144dpi.
